README.rst

```diff
@@ -33,6 +33,13 @@ Example
     # prints (['en', 'de'], array([0.50208992, 0.30427793]))
     print(identify_language('And then he said "Ich liebe dich"!', with_prob=True, k=2))
 
+    # if you want to use your custom model
+    print(identify_language('Hello, how are you?', model_path='model.ftz'))
+
+    # if you don't want to load it on every request
```

Comment:
@swecooo this does not seem to be technically true -- your other change with `DEFAULT_MODEL` makes it so that the model won't be loaded on every request, right?

How about something like this:

```suggestion
    # if you would like to pass the model yourself
```

Reply:
Yes, you're right, it's ambiguous. Basically if you pass the `model_path` it will get loaded on every request, however if you don't provide it, the `DEFAULT_MODEL` is already pre-loaded. So it's true in one case and not in the other.

What do you think about following?

```suggestion
    # if you don't want to load it on every request
    # if you would like to pass the model yourself or prevent from it being loaded on every request
```

Reply:
I do not seem to have permissions to your commit the suggestion to your repository but I would be totally OK with what you suggested -- feel free to update it and resolve this conversation.